### PR TITLE
Update 11-RenatSeredzhinov.md

### PR DESCRIPTION
**Pull request by RenatSeredzhinov**
- [x] First

### CODE REVIEW

File: _slides/11-RenatSeredzhinov.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Спартак -чемпион!
```
Comment (gusshawstewart) [inline _slides/11-RenatSeredzhinov.md:6]: This is great work! 😄 
Спасибо!

File: _slides/11-RenatSeredzhinov.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (gusshawstewart) [inline _slides/11-RenatSeredzhinov.md]: Is this better?
```suggestion
title: "Welcome to our slide deck! It is the best slide deck in the world!"
```

**Review by gusshawstewart — APPROVED** [review]
Looks Good To Me! 🚢 🥇

File: _slides/11-RenatSeredzhinov.md (hunk 1 above)
Comment (RenatSeredzhinov) [reply _slides/11-RenatSeredzhinov.md:6]: Thanks )))

**Pull request merged**